src/main/pages/che-7/end-user-guide/assembly_using-gradle-artifact-repositories.adoc

```diff
@@ -0,0 +1,32 @@
+---
+title: Using Gradle artifact repositories
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/using-gradle-artifact-repositories/
+folder: che-7/end-user-guide
+summary:
+---
+
+:page-liquid:
+
+:parent-context-of-using-maven-artifact-repositories: {context}
+
+[id="using-gradle-artifact-repositories_{context}"]
+= Using Gradle artifact repositories
+
+:context: using-gradle-artifact-repositories
+
+To use Gradle in a restricted environment, you must configure a Gradle Wrapper script which handles the different versions of Gradle for your projects and an initialization script which adds the Gradle artifact repository to every Gradle project in your workspace.
```

Comment:
This section seems to have little to do with the text I wrote.

Reply:
Removed.